fixtures/questions/get_question_fixtures.py

```diff
@@ -0,0 +1,26 @@
+get_question_query = '''{
+  feedbackQuestion {
+    totalResponses
+    responses {
+      roomName,
+      responseCount,
+      cleanlinessRating
+    }
+  }
+}
+'''
+
+get_question_query_response = {
+  "data": {
+    "feedbackQuestion": {
```

Comment:
I am of the idea that we could get rid of this additional nesting created at the feedbackQuestion so that we can directly move from data to totalResponses and then responses, what do you think?

Reply:
That is a good idea. I will make the changes.